packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -118,9 +118,20 @@ def test_pod_logs(cluster):
 
     with cluster.marathon.deploy_pod_and_cleanup(pod_definition):
         url = get_task_url(cluster, pod_id)
+        url_parts = url.split('/')
+        assert url_parts, 'Invalid URL {}'.format(url)
```

Comment:
please make this assert more specific. url.split will return a list always a list so this will never raise an assertion error. Doing ```assert foo``` is only valid when you are checking for foo being a zero length string or None